elasticsearch/etc/elasticsearch/elasticsearch-data.yml

```diff
@@ -1,4 +1,13 @@
 {%- set master_fqdn = salt['mine.get']('G@stack_id:' ~ grains.stack_id ~ ' and G@roles:elasticsearch.master', 'grains.items', 'compound').values()[0]['fqdn'] -%}
+{%- set num_shards = ((salt['mine.get']('G@stack_id:' ~ grains.stack_id ~ ' and G@roles:elasticsearch.data', 'grains.items', 'compound') | length) / 2) | int -%}
+{%- if num_shards < 5 -%}
+    {%- set num_shards = 5 -%}
+{%- endif -%}
+
+{%- set num_replicas = pillar.elasticsearch.replicas -%}
```

Comment:
@cpenner461 it has a default of 2 in the SPECFILE, so that will get used on blueprints where's it's not specified.